input/resources/valueset-common-au-recorded-sex-or-gender-source-document-jurisdiction.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="common-au-recorded-sex-or-gender-source-document-jurisdiction" />
+    <meta>
+        <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset" />
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+        <valueInteger value="0"/>
+    </extension>
+    <url value="http://terminology.hl7.org.au/ValueSet/common-au-recorded-sex-or-gender-source-document-jurisdiction" />
```

Comment:
url is too long - it is not an exact copy of the title. Change this, and the id.

Reply:
Done.